streams/src/test/java/org/apache/kafka/streams/processor/internals/StateDirectoryTest.java

```diff
@@ -101,17 +102,21 @@ public class StateDirectoryTest {
     private File appDir;
 
     private void initializeStateDirectory(final boolean createStateDirectory, final boolean hasNamedTopology) throws IOException {
+        initializeStateDirectory(createStateDirectory, hasNamedTopology, false);
+    }
+
+    private void initializeStateDirectory(final boolean createStateDirectory, final boolean hasNamedTopology,
+            final boolean allowOsGroupWriteAccess) throws IOException {
```

Comment:
nit: formatting -- we usually either keep all parameter in a single line, or make one line per parameter. The current formatting is kinda hard to read.